pkg/status/clusteragent/status_apiserver.go

```diff
@@ -1,11 +1,11 @@
 // Unless explicitly stated otherwise all files in this repository are licensed
 // under the Apache License Version 2.0.
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
-// Copyright 2016-present Datadog, Inc.
+// Copyright 2023-present Datadog, Inc.
```

Comment:
Just to make sure: this is on purpose and the file wasn't created before 2023 ?
(same question for the other files were you changed the copyright year)

FYI 4200 Go files out of 4900 have a copyright with year 2016 so I don't think it's a big deal to leave 2016 :smile: